### ISSUE TITLE
Update to the new API

### ISSUE THREAD

**Issue by pieterzen** [thomascys/solaxcloud#8]
If at all possible, we could rebuild this integration to use the new API. There are reports of login failures, but that's probably related to the latest HA 117 build.  

The new API is documented (in the Solax Cloud portal) and uses a generated token instead of username/password. 

> User can get a specific range of information through the granted tokenID. Please
> obtain your tokenID on the API page of Solaxcloud for free.
> The tokenID can be used to obtain real-time data of your inverter system. The
> obtain frequency need to be lower than 10 times/min and 10,000 times/day.

**Comment by pieterzen** [comment]
Yeah, well, great story from the dev at Solax, but the data is still only updated every 5min, so there is no use to make any change.

**Comment by thomascys** [comment]
@pieterzen thanks for the information.
The tokenID is still a better approach than the username/password combination.
I'll try to look into it when i have some spare time.

**Comment by pieterzen** [comment]
The tokenId sounds like a good idea so there is no login but the data that
comes back from that new call isn't as complete.
While testing yesterday I've noticed that my old tokenId (the one you get
after login) is still valid. So in a way we already have a persisting
token.
Therefor I feel that there is no need to use the new getRealtimeInfo IF you
know a way to store the tokenId after the first login. You think that's
possible?

Regarding the decimals, I believe we should return Watt and WattHour (so
divide by 1000). That would remove the need for rounding.
